packages/builder/src/stores/portal/licensing.ts

```diff
@@ -133,6 +136,16 @@ class LicensingStore extends BudiStore<LicensingState> {
     return aiCredits > aiCreditsLimit
   }
 
+  actionsExceeded(
+    actions: number,
+    actionsLimit = get(this.store).actionsLimit
+  ) {
+    if (actionsLimit === UNLIMITED || actionsLimit === undefined) {
```

Comment:
NAB: I don't think `actionsLimit === undefined` is needed because `undefined > actionsLimit` will always be false.
```suggestion
    if (actionsLimit === UNLIMITED) {
```